modules/telco-core-security.adoc

```diff
@@ -1,31 +1,68 @@
++
+--
+Telco customers are security conscious and require clusters to be hardened against multiple attack vectors.
 In {product-title}, there is no single component or feature responsible for securing a cluster.
 Use the following security-oriented features and configurations to secure your clusters:
 
 * **SecurityContextConstraints (SCC)**: All workload pods should be run with `restricted-v2` or `restricted` SCC.
-* **Seccomp**: All pods should be run with the `RuntimeDefault` (or stronger) seccomp profile.
-* **Rootless DPDK pods**: Many user-plane networking (DPDK) CNFs require pods to run with root privileges. With this feature, a conformant DPDK pod can be run without requiring root privileges.
+* **Seccomp**: All pods should run with the `RuntimeDefault` (or stronger) seccomp profile.
+* **Rootless DPDK pods**: Many user-plane networking (DPDK) CNFs require pods to run with root privileges.
+With this feature, a conformant DPDK pod can run without requiring root privileges.
 Rootless DPDK pods create a tap device in a rootless pod that injects traffic from a DPDK application to the kernel.
-* **Storage**: The storage network should be isolated and non-routable to other cluster networks. See the "Storage" section for additional details.
+* **Storage**: The storage network should be isolated and non-routable to other cluster networks.
+See the "Storage" section for additional details.
+
+Refer to link:https://access.redhat.com/articles/7090422[Custom nftable firewall rules in OpenShift] for a supported method of implementing custom nftables firewall rules in OpenShift cluster nodes.
+This article is intended for cluster administrators who are responsible for managing network security policies in OpenShift environments.
+
+It is crucial to carefully consider the operational implications before deploying this method, including:
+
+* **Early application**: The rules are applied at boot time, before the network is fully operational.
+Ensure the rules don't inadvertently block essential services required during the boot process.
+* **Risk of misconfiguration**: Errors in your custom rules can lead to unintended consequences, potentially leading to performance impact or blocking legitimate traffic or isolating nodes.
+Thoroughly test your rules in a non-production environment before deploying them to your main cluster.
+* **External endpoints**: OpenShift requires access to external endpoints to function.
+For more information about the firewall allowlist, see "Configuring your firewall for {product-title}".
+
+Ensure that cluster nodes are permitted access to those endpoints.
+* **Node reboot**: Unless node disruption policies are configured, applying the `MachineConfig` CR with the required firewall settings causes a node reboot.
+Be aware of this impact and schedule a maintenance window accordingly.
+For more information, see "Using node disruption policies to minimize disruption from machine config changes".
```

Comment:
This xref, ""Using node disruption policies to minimize disruption from machine config changes"", is no where ot be found.